src/components/ProfilePage/Bio.jsx

```diff
@@ -54,27 +54,32 @@ class Bio extends React.Component {
         expression={editMode}
         cases={{
           editing: (
-            <Form onSubmit={this.handleSubmit}>
-              <FormGroup>
-                <Label for={formId}>{intl.formatMessage(messages['profile.bio.about.me'])}</Label>
-                <Input
-                  type="textarea"
-                  id={formId}
-                  name={formId}
-                  value={bio || ''}
-                  invalid={error != null}
-                  onChange={this.handleChange}
+            <div role="dialog" aria-labelledby={`${formId}-label`}>
```

Comment:
The British spelling of "labelled" always makes me do a double take.  Better than "Referer", I guess.  :-P

Reply:
Yea, I only know it's correct if it's the right colour in the IDE ;)

Reply:
I see what you did there!